.github/actions/setup-sentry/action.yml

```diff
@@ -30,7 +30,7 @@ inputs:
   python-version:
     description: 'python version to install'
     required: false
-    default: '3.8.16'
+    default: '3.8.17'
```

Comment:
this version has known breakages for sentry-sdk so we can't do this quite yet -- https://github.com/getsentry/sentry-python/issues/2160#issuecomment-1581120574

Reply:
The breakage on the SDK side means we can't upgrade on the server side?

Also, I'm seeing a fix for the SDK shipped [in 1.25.1](https://github.com/getsentry/sentry-python/releases/tag/1.25.1). Not enough?

Reply:
(1) yes, we use our own sdk
(2) correct, [we are not on that version](https://github.com/getsentry/sentry/blob/ce47e0375660c20d5a01047ec2961af20986c490/requirements-frozen.txt#L123)

Reply:
no, those should be two separate PRs of two distinct actions rather than coupling two unrelated changes

Reply:
should i create the Pr for sentry SDK version upgrade.